config.go

```diff
@@ -207,6 +207,9 @@ func xmlToFileLogWriter(filename string, props []xmlProperty, enabled bool) (*Fi
 	}
 
 	flw := NewFileLogWriter(file, rotate)
+	if flw == nil {
```

Comment:
This prevents the stack trace. Unfortunately log4go still insists on exiting directly rather than returning any kind of error - I'll make a ticket to handle that.